src/domain/tokens/entities/schemas/token-info.schema.ts

```diff
@@ -0,0 +1,15 @@
+import { JSONSchemaType } from 'ajv';
+import { Token, TokenType } from '../token.entity';
+
+export const tokenSchema: JSONSchemaType<Token> = {
+  type: 'object',
+  properties: {
+    address: { type: 'string' },
+    decimals: { type: 'number' },
+    logoUri: { type: 'string' },
```

Comment:
Since we know that this is a `uri` we can do something like:

```typescript
logoUri: { type: 'string', format: 'uri' }
```

Reply:
Fixed in [134ec39](https://github.com/5afe/safe-client-gateway-nest/pull/178/commits/134ec39812f27abc6b233638507d224fe415e9a1)